tracing/src/lib.rs

```diff
@@ -174,6 +174,15 @@
 //! # fn main() {}
 //! ```
 //!
+//! For external functions which don't have built-in tracing support and can't have
+//! the `#[instrument]` attribute applied, the [`Span` struct][`Span`] has a
+//! [`in_scope()` function][`in_scope`] which can be used to easily wrap synchonous
+//! code in a span.
+//!
+//! For example:
```

Comment:
take it or leave it: maybe we ought to also have an example showing that `in_scope` can be called multiple times on the same span, like
```rust
let span = tracing::info_span!(...);

let something = span.in_scope(|| some_function());
let other_thing = span.in_scope(|| other_function());
```
or whatever?

On the other hand, maybe this is better left in the function-level docs for `in_scope`?

Reply:
I think we should leave that in the `in_scope()` docs. Even just mentioning it here would probably be enough, but I think the serde example will be quite nice.